packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -483,31 +505,112 @@ export class PerformanceEvent {
     }
 }
 
+// Note - these Telemetry types should move to common-definitions package
+
+/**
+ * Broad classifications to be applied to individual properties as they're prepared to be logged to telemetry.
+ * Please do not modify existing entries for backwards compatibility.
+ */
+export enum TelemetryDataTag {
+    /** Non-sensitive data that can be logged plainly */
+    None = "None",
```

Comment:
I'd find better (more descriptive) name for None. If I see for first time it in the code, I do not know what to think about how it's handled. I think name should clearly reflect expectations - misunderstandings here are too expensive.

Reply:
That said, do we need it? That's the default handling, and I'd expect all the code not to use ITaggedTelemetryPropertyType for that, but rather use TelemetryEventPropertyType instead. I.e. maybe tagged version is only if there is a potential info disclosure only?

Reply:
Ok my instinct is that it's good to have the option to be explicit, but both you and Tony suggested removing this. I totally agree that the incoming type should always support unwrapped primitives, but maybe I've had in mind the idea of always tagging the outgoing type for simpler processing by the host logger.  But then that can be achieved without a default tag by making `tag` optional on `ITaggedTelemetryPropertyType`. I'll remove it.